### PR TITLE
[AIP-140] Add rule prohibiting prepositions.

### PR DESCRIPTION
**Pull request by lukesneeringer**

### CODE REVIEW

File: rules/aip0140/prepositions.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+	"github.com/scylladb/go-set/strset"
+)
+
+var noPrepositions = &lint.FieldRule{
+	Name: lint.NewRuleName("core", "0140", "prepositions"),
+	URI:  "https://aip.dev/140#prepositions",
+	LintField: func(f *desc.FieldDescriptor) (problems []lint.Problem) {
+		banned := strset.New(
```
Comment (mingzhi) [inline rules/aip0140/prepositions.go]: I am not sure if using `go-set` package would save more time than importing and maintaining the package in Google3. The package is currently not in Google3.

Comment (lukesneeringer) [reply rules/aip0140/prepositions.go]: Oh, ugh. That makes sense.
Is there an equivalent package that _is_ imported into google3?

Comment (mingzhi) [reply rules/aip0140/prepositions.go]: `bitbucket.org/creachadair/stringset` is a very popular one (>1000 imports) in Google3.

Comment (lukesneeringer) [reply rules/aip0140/prepositions.go]: Sold, thanks!

Comment (lukesneeringer) [reply rules/aip0140/prepositions.go]: Done. Thanks.

**Pull request merged**